src/broker/Manager.cc

```diff
@@ -355,7 +355,7 @@ std::string Manager::NodeID() const
 	return to_string(bstate->endpoint.node_id());
 	}
 
-bool Manager::PublishEvent(string topic, std::string name, broker::vector args)
+bool Manager::PublishEvent(const string& topic, const std::string& name, broker::vector args)
```

Comment:
There's actually a ton of complaints about those moves as a result of the `
performance-move-const-arg` checker, but I wasn't going to touch them in this PR. I can definitely add them in though.

Reply:
I'm going to back out the `broker/Manager` changes so that I can focus on the const and move issues all together in one PR later.

Reply:
Sure, but also was saying there's possibly no issue in the first place.  Taking arguments by const-ref here means a copy is going to happen when creating a CAF message, which utilizes message passing as part of the concurrency model.  It's not too nice to share references/pointers across threads since that becomes easy to miss-manage, so the expectation is usually either move or copy the data that needs to be part of the message.  So the by-value args here means we can potentially forward the data all the way into the creation of a message via `std::move()` and avoid an unnecessary copy -- it's just up to the caller to `std::move()` their arguments when/if they can otherwise the single copy will just happen at that point of call (rather than the point of message creation, as would be necessary with const-ref args).